src/controller.rs

```diff
@@ -153,6 +159,18 @@ impl Controller for FoxBox {
         format!("ws://{}:{}/services/{}/", self.hostname, self.ws_port, service_id)
     }
 
+    fn get_config(&self, namespace: String, property: String, default: String)
+        -> String {
+        match self.config.read().unwrap().get(namespace, property) {
```

Comment:
It only fails on poisoned locks which only occur when a writer panics. If that happens, there is not much sense in trying to recover here, is it?